content/en/docs/rosenpass-tool/guides/debian.md

```diff
@@ -0,0 +1,275 @@
+
+The command also install `curl` in preparation of the Rustup installation in the next point.
+
+<span class="codebox">
+
+```sh{class="command-root"}
+apt --yes install libsodium-dev libclang-dev cmake pkg-config git build-essential curl wireguard
+```
+
+</span>
+
+2. **Install** Rust. The preferred way to do this is by using [Rustup](https://rustup.rs/). During installation, Rustup asks you about your installation preferences; if you are unsure you can just proceed with the standard installation. If possible, install the latest release. In July 2023, the minimum required Rust version for Rosenpass was 1.64.0.
+<span class="spacer"></span>
+
+3. **Clone** the Rosenpass Git repository, change to the `rosenpass` directory, and switch to the branch of the latest release:
+
+```sh{class="code-block-list command-user"}
+git clone https://github.com/rosenpass/rosenpass.git
+```
+```sh{class="code-block-list command-user"}
+cd rosenpass
+```
+```sh{class="code-block-list command-user"}
+git checkout v0.2.2
+```
+
+<span class="spacer"></span>
+Without switching to a different branch, you will install and compile the development version. If you would like to compile an ancient version, find its Git tag name by browsing the [releases page](https://github.com/rosenpass/rosenpass/releases) and use `git checkout` accordingly.
+<span class="spacer"></span>
+
+4. **Compile:** This is the actual compilation step and may take a while:
+
+<span class="codebox">
+
+```sh{class="command-user"}
+cargo build --release
+```
+
+</span>
+
+5. **Install:** If the compilation succeeded, proceed to install Rosenpass by copying its binaries to `/usr/local/bin` **with root privileges**:
+
+```sh{class="code-block-list command-root"}
+install target/release/rosenpass /usr/local/bin
+```
+```sh{class="code-block-list command-root"}
+install rp /usr/local/bin
+```
+<span class="spacer"></span>
+The second binary, `rp`, is the Rosenpass helper. You can skip this step if you do not want to use it.
+<span class="spacer"></span>
+
+**That's it! You have downloaded, compiled, and installed Rosenpass.** You can now proceed to [set up a Rosenpass-enhanced WireGuard VPN](#set-up-a-rosenpass-enhanced-wireguard-vpn).
+
+<span class="spacer"></span>
+
+{{% /blocks/section %}}
+
+
+{{< blocks/lead color="secondary" class="title-box" >}}
+## Set up a Rosenpass-enhanced WireGuard VPN
+{{< /blocks/lead >}}
+
+{{% blocks/section color="light" class="no-flex contains-code-snippets compilation" %}}
+
+In this section, we set up a Rosenpass-enhanced WireGuard connection between two peers. Technically, there's no difference between the two peers. However, for clarity, we name them <span class="primary-highlight"><strong>server</strong> (pink)</span> and <span class="secondary-highlight"><strong>client</strong> (orange)</span>.
+
+### Prepare the Key Pairs
```

Comment:
These steps seem to be the same for every linux, should we put this into a template and include that template in all Linux pages?

Reply:
Yes, we should definitely make this more modular. I am tempted to leave it like this for now as a “make it work” step, and then, to “make it beautiful”, transition to something like UV does https://docs.astral.sh/uv/getting-started/installation/#__tabbed_1_1 where everything is on one page and the visitor can click on a tab to choose between different operating systems. On the UV example, I like that all tabs on the page change when one of them is clicked.

Reply:
I added an issue for this: #135